backend/index.js

```diff
@@ -0,0 +1,35 @@
+import fs from "fs";
+import express from "express";
+
+const app = express();
+
+let sessionCounter = 0;
+let counter;
+
+try {
+  fs.existsSync("temp/my-counter-visited-times.txt")
+    ? (counter = fs.readFileSync("temp/my-counter-visited-times.txt", "utf8"))
+    : (counter = 0);
+} catch (e) {
+  console.log(e);
+}
+
+app.get("/", (req, res) => {
+  sessionCounter++;
+  counter = parseInt(counter);
+  res.send(
+    `This page was visited ${
+      counter + sessionCounter
+    } times in total, ${sessionCounter} out of which was during the current code execution session ( statuscode:${
+      res.statusCode
```

Comment:
Ah I see you did it like this but there is no need to send it in the message as the FE user can always wee the status of the request by opening the network tab in dev tools. PTAL at the comment above.